### DISCUSSION TITLE
2019/05/combining-calendars-across-outlook-and-google/

### DISCUSSION THREAD

**Discussion by giscus** [SeanKilleen/seankilleen.github.io#751] [Specific Post Comments]
# 2019/05/combining-calendars-across-outlook-and-google/

This came up in a conversation and I realized my methods might be helpful for someone. I figured I can write it up with step-by-steps here.

https://seankilleen.com/2019/05/combining-calendars-across-outlook-and-google/

**Comment by navnoot** [comment]
I just made a github account to comment on this. THANKYOU. I've had this issue for 6 months with no way to solve it until that little app you mentioned, didn't work due to permission problems the first time but I just restarted it and it worked like a charm!! I love you, now I don't need to juggle my med school and personal calendars anymore.
